jest.config.mjs

```diff
@@ -7,5 +7,5 @@ export default {
       useESM: true,
     }],
   },
+  testTimeout: 50000,
```

Comment:
jesus that's a long test :p

Reply:
The local ganache integration takes some time to start and stop. Worth revisiting and seeing if it can be switched to using Anvil programmatically.